sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/SessionProcessorLiveTests.cs

```diff
@@ -779,5 +777,191 @@ Task ExceptionHandler(ProcessErrorEventArgs args)
             }
         }
 
+        [Test]
+        [TestCase(1, 2, false)]
+        [TestCase(5, 3, true)]
+        [TestCase(10, 10, false)]
+        [TestCase(20, 10, false)]
```

Comment:
can we throw in a few more test cases? Maybe one where num threads = 20, session count = 40. Also numthreads = 1 and session count = 1 would be good.